src/main/groovy/org/mockito/release/gradle/ReleaseConfiguration.java

```diff
@@ -387,23 +386,6 @@ public void setDevelopers(Collection<String> developers) {
         public void setContributors(Collection<String> contributors) {
             configuration.put("team.contributors", contributors);
         }
-
-        /**
-         * A boolean flag for fetch all contributors from GitHub to include them in generated pom file.
-         * This is optional value, by default set to true.
-         * <p>
-         * See POM reference for <a href="https://maven.apache.org/pom.html#Contributors">Contributors</a>.
-         */
-        public boolean isAddContributorsToPomFromGitHub() {
```

Comment:
Yes, the use case is important and we satisfy it: when releasing.team.contributors has any elements, we don't fetch contributors from github and don't populate releasing.team.contributors. Thoughts?